lib/color.rb

```diff
@@ -1,7 +1,17 @@
 # Colorization for strings
 class String
+  @use_color = STDOUT.isatty
+
+  class << self
+    attr_accessor :use_color
+  end
+
   def colorize(color_code, shade)
-    "\e[#{shade};#{color_code}m#{self}\e[0m"
+    if self.class.use_color
```

Comment:
It would be easier to use an `@` variable.

Reply:
I had two reasons for doing it this way:
1) My thinking was that `String#colorize` shouldn't need any knowledge outside of the class in which it is defined. I didn't want to have to pass an additional parameter to every coloring method, so a class instance variable seemed to fit the bill perfectly.
2) I don't really understand variable scope in ruby. So far I've just been mimicking nearby code, so I wasn't sure how to use an `@` variable and have it be visible in crew.rb. I really need to take a weekend and start learning ruby for real 😁

Reply:
The `@` variable makes it so you don't need to pass the variable.  It's basically a global scope within the class.

Reply:
OK, I think I understand what is happening here.  You cannot use the `@` variable since the scope is from another class.